bin/instrumented_uwsgi.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+#
+# Run uwsgi + Open Zaak in profiling mode.
+#
+# Run from the root of the repository:
+#
+#   ./bin/profile.sh
+# 
+
+set -ex
+
+: ${DB_HOST:=localhost}
+: ${DB_PORT:=5432}
+: ${DB_NAME:=openzaak-perf}
+
+: ${UWSGI_PROCESSES:=4}
+: ${UWSGI_THREADS:=4}
+: ${UWSGI_MASTER:=1}
+
+# Fixed settings to enable profiling with Silk & get all the necessary instrumentation.
+export \
+    DEBUG=false \
+    DJANGO_SETTINGS_MODULE=openzaak.conf.dev \
+    PROFILE=true \
```

Comment:
enabling profiling already causes overhead, so for raw metrics, disable all profiling (at most you can keep some `timeit.default_timer()` print statements